guides/micronaut-data-jdbc-repository/src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+micronaut:
+  application:
+    name: micronautguide
+---
+#tag::datasource[]
+datasources:
+  default:
+    url: jdbc:mysql://localhost:3306/db
+    driverClassName: com.mysql.cj.jdbc.Driver
+    username: root
+    password: ''
```

Comment:
The original `application.yml` showed: 

```
    url: ${JDBC_URL:`jdbc:h2:mem:default;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE`}
     username: ${JDBC_USER:sa}
     password: ${JDBC_PASSWORD:""}
     driverClassName: ${JDBC_DRIVER:org.h2.Driver}
```

which I think it is good. Because we were teaching the user how to avoid hardcoding credentials in their configuration files and use environment variables instead.